<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemporalProjectionViewPropertiesWidget</class>
 <widget class="QWidget" name="TemporalProjectionViewPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temporal Projection View Properties</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <!-- Point Data Keys Section -->
   <item>
    <widget class="QLabel" name="point_data_label">
     <property name="text">
      <string>Point Data Keys:</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="point_data_table">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Data Key</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="add_point_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="add_point_layout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="add_point_combo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_point_button">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="remove_point_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <!-- Line Data Keys Section -->
   <item>
    <widget class="QLabel" name="line_data_label">
     <property name="text">
      <string>Line Data Keys:</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="line_data_table">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Data Key</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="add_line_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="add_line_layout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="add_line_combo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_line_button">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="remove_line_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <!-- Rendering Options Section -->
   <item>
    <widget class="QLabel" name="rendering_label">
     <property name="text">
      <string>Rendering:</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rendering_options_widget" native="true">
     <layout class="QFormLayout" name="rendering_form_layout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="point_size_label">
        <property name="text">
         <string>Point Size:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="point_size_spinbox">
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>50.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="line_width_label">
        <property name="text">
         <string>Line Width:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="line_width_spinbox">
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <!-- Selection Mode Section -->
   <item>
    <widget class="QLabel" name="selection_label">
     <property name="text">
      <string>Selection Mode:</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="selection_mode_combo">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Point</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Line</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="clear_selection_button">
     <property name="text">
      <string>Clear Selection</string>
     </property>
    </widget>
   </item>
   <!-- Spacer -->
   <item>
    <spacer name="vertical_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
